core/domain/rte_component_registry_test.py

```diff
@@ -198,6 +203,10 @@ def test_require_file_contains_all_imports(self):
 
         rtc_ts_filenames = []
         for component_id in feconf.ALLOWED_RTE_EXTENSIONS:
+            # TODO(#9356): Remove this if condition once the Svgeditor
+            # directves is created in the second part of 1st milestone.
```

Comment:
"directive" is misspelled, here and above.

Reply:
Done.